MathLib/LinAlg/PETSc/PETScVector.h

```diff
@@ -217,7 +215,7 @@ class PETScVector
          * This method is dangerous insofar as you can do arbitrary things also
          * with a const PETSc vector!
          */
-        PETSc_Vec getRawVector() const {return _v; }
+        PETSc_Vec getRawVector() const { return _v; }
```

Comment:
Is getData(), which is "TODO eliminate" above, still needed, or can it be replaced with getRawVector() now?